renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "dependencyDashboard": true,
+  "pinVersions": true,
+  "vulnerabilityAlerts": {
+    "groupName": "security updates",
+    "enabled": true,
+    "schedule": [],
```

Comment:
what does this mean?

why include this default value and some others but not all?

Reply:
Yes, I initially added schedule: [] "default value is as soon as vulnerabilities are found" with the intention to specify a duration. i will remove it.

As for the other values, I didn’t add them because I assumed the default settings would meet our needs.

Reply:
what is the `enabled` and `prCreation` default value?

Reply:
prCreation default"immediate" [here](https://docs.renovatebot.com/configuration-options/#prcreation)
enabled default"true" [here](https://docs.renovatebot.com/configuration-options/#enabled)

Reply:
then why do we set the same here explicitly?